xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,645 @@
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replica ID</term>
+    <listitem>
+     <para>
+      A unique identifier of the &ds389; instance within the replication
+      topology.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replication manager</term>
+    <listitem>
+     <para>
+      An account with replication rights in the directory.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect2>
+
+ <sect2 xml:id="sec-security-ldap-replication-configuration">
+  <title>Configuring replication</title>
+  <para>
+   The first example sets up a two node bi-directional replication with a
+   single read-only server, as a minimal starting example. In the following
+   examples, the host names of the two read-write nodes are RW1 and RW2, and
+   the read-only server is RO1. (Of course you must use your own host names.)  
+   </para>
+  <para>
+   All servers should have a backend with an identical suffix. Only one server,
+   RW1, needs an initial copy of the database.
+  </para>
+  <important>
+  <title>Create a strong replication manager password</title>
+  <para>
+    The replication manager should be considered equivalent to the 
+    directory manager, in terms of security and access, and should
+    have a very strong password.
+  </para>  
+  <para>
+    If you create different replication manager passwords for each server,
+    be sure to keep track of which password belongs to which server. For
+    example, when you configure the outbound connection in RW1's replication agreement, 
+    you need to set the replication manager password to the RW2 replication manager password.
+  </para>
+</important>
+  
+ <sect3 xml:id="sec-security-ldap-replication-two-nodes">  
+   <title>Configuring two-node replication</title>
+  <para>
+    The following commands configure the read-write replicas in a two-node setup 
+    (<xref linkend="ex-ldap-replication-two-replicas"/>), with the hostnames
+    RW1 and RW2. (Remember to use your own hostnames.) First, configure RW1:
+  </para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication enable</command> \ 
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
+<command>--role supplier --replica-id 1 --bind-dn "cn=replication manager,cn=config"</command>
+</screen>
+  <para>
+   Configure RW2:
+  </para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication enable</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \ 
+<command>--role supplier --replica-id 2 --bind-dn "cn=replication manager,cn=config"</command>
```

Comment:
--replica-id 1 and 2 need to be replacable.

Reply:
:+1: